src/main/java/com/cx/plugin/cli/constants/Command.java

```diff
@@ -153,6 +153,16 @@ public static Options getOptions() {
 
         options.addOption(NTLM, false, ArgDescriptions.NTLM);
 
+        options.addOption(ENV_VARIABLE, true, ArgDescriptions.ENV_VARIABLE);
+        options.addOption(SAST_PROJECT_ID, true, ArgDescriptions.SAST_PROJECT_ID);

```

Comment:
AstScaClient gets the projectId from project name.  Check with Product manager if we need separate project Id prompt.